tango_ros_common/tango_ros_native/include/tango_ros_native/tango_ros_node.h

```diff
@@ -90,7 +95,12 @@ const std::string SAVE_MAP_SERVICE_NAME = "save_map";
 const std::string CONNECT_SERVICE_NAME = "connect";
 
 const std::string DATASETS_PATH = "/sdcard/tango_ros_streamer/datasets/";
-const double TANGO_3D_RECONSTRUCTION_DEFAULT_RESOLUTION = 0.05; // meter
+const double TANGO_3DR_DEFAULT_RESOLUTION = 0.05; // meter
+const bool TANGO_3DR_DEFAULT_USE_SPACE_CLEARING = false;
+const int32_t TANGO_3DR_DEFAULT_MIN_NUM_VERTICES = 1;
+const int32_t TANGO_3DR_DEFAULT_UPDATE_METHOD = 0; // TRAVERSAL_UPDATE
+const int32_t TANGO_3DR_DEFAULT_MAX_VOXEL_WEIGHT = 16383; // Roughly number of observations.
```

Comment:
This comes straight from the Tango Developers documentation: "The maximum voxel weight for the integration filter. Defaults to 16383. Roughly corresponds to the number of observations. A higher value gives better geometric quality and stability; a lower value gives more responsiveness to dynamic objects."